packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -125,6 +125,8 @@ export interface DropdownProps
    */
   itemToString?: (item: ShorthandValue) => string
 
+  labelId?: string
```

Comment:
do we need this prop? can't we just use 'aria-labelledby' attribute directly?

It also misses documentation

Reply:
Oups, will add documentation!

Maybe because we need to compute `aria-labelledby` from that (in case of the trigger button, we also add its own id). Also because it is used in two places: button/input and ul. I think it's better this way. We will have it computed in the behavior in the future. Will ask for more opinions on this.

Reply:
ok, let's see what others say
@layershifter @kuzhelov @mnajdova